source/change-streams/tests/change-streams.yml

```diff
@@ -297,3 +297,163 @@ tests:
           fullDocument:
             z:
               $numberInt: "3"
+  -
+    description: Test insert, update, replace, and delete event types
+    minServerVersion: "3.6.0"
+    target: collection
+    topology:
+      - replicaset
+    changeStreamPipeline: []
+    changeStreamOptions: {}
+    operations:
+      -
+        database: *database_name
```

Comment:
@kvwalker: I think it's worth pointing out that this deviates from `database_name` being used in the CRUD tests (https://github.com/mongodb/specifications/pull/437#discussion_r243255739). If you decide to keep the operation-level DB name in those tests, it may be worth standardizing with this format.

Reply:
Per https://github.com/mongodb/specifications/pull/437, the CRUD tests no longer have a `database` field within an `operation` element, so that's no longer a concern.

It looks like this field is still necessary because the `target` field only specifies the type of object (i.e. client, database, collection) and we need this to denote the name of the corresponding object to invoke `watch()`. In `changes-streams.yml`, this juggles between `*database_name` and `*database2_name`. Do you concur that this is still necessary? If so, we can resolve.

Reply:
Yes, it's still necessary.